README.md

```diff
@@ -109,6 +109,12 @@ As mentioned, the goal of this library is to be generic over a QUIC implementati
 This crate as well as the quic implementations are tested ([quinn](https://github.com/quinn-rs/quinn-interop), [s2n-quic](https://github.com/aws/s2n-quic/tree/main/scripts/interop)) for interoperability and performance in the [quic-interop-runner](https://github.com/marten-seemann/quic-interop-runner).
 You can see the results at (https://interop.seemann.io/).
 
+## Debugging
```

Comment:
Since this is only relevant to the examples, I'd put it in the `examples/README.md` instead.